v0.13.0 crash on android-2.3

With version 0.13.0 i SMSSecrure don't start (in French "Désolé Fermeture soudaine  de l'application SMSSecure").

Device info:
```
build.board: spade
build.bootloader: 2.00.0027
build.brand: vodafone_fr
build.cpu_abi: armeabi-v7a
build.cpu_abi2: armeabi
build.device: ace
build.display: GRJ90
build.fingerprint: vodafone_fr/htc_ace/ace:2.3.5/GRJ90/208029.3:user/release-keys
build.hardware: spade
build.host: ABM001
build.id: GRJ90
build.manufacturer: HTC
build.model: HTC Desire HD A9191
build.product: htc_ace
build.radio: unknown
build.serial: HT0AWRX04304
build.tags: release-keys
build.time: 1321968286000
build.type: user
build.user: unknown
version.codename: REL
version.incremental: 208029.3
version.release: 2.3.5
version.sdk_int: 10
```

Crash (some in both logs):
```
01-09 08:17:57.319 D/Database( 2896): dbopen(): path = /data/data/org.smssecure.smssecure/databases/canonical_address.db, mode: wal, disk free size: 223 M, handle: 0x36af78
01-09 08:17:57.319 D/dalvikvm( 2874): GC_FOR_MALLOC freed <1K, 43% free 4109K/7111K, external 0K/0K, paused 44ms
01-09 08:17:57.349 E/AndroidRuntime( 2896): FATAL EXCEPTION: main
01-09 08:17:57.349 E/AndroidRuntime( 2896): java.lang.RuntimeException: Unable to start activity ComponentInfo{org.smssecure.smssecure/org.smssecure.smssecure.IntroScreenActivity}: java.lang.NullPointerException
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1872)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1893)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at android.app.ActivityThread.access$1500(ActivityThread.java:135)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1054)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at android.os.Handler.dispatchMessage(Handler.java:99)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at android.os.Looper.loop(Looper.java:150)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at android.app.ActivityThread.main(ActivityThread.java:4385)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at java.lang.reflect.Method.invokeNative(Native Method)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at java.lang.reflect.Method.invoke(Method.java:507)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:849)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:607)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at dalvik.system.NativeStart.main(Native Method)
01-09 08:17:57.349 E/AndroidRuntime( 2896): Caused by: java.lang.NullPointerException
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at com.nineoldandroids.animation.PropertyValuesHolder.setupSetterAndGetter(PropertyValuesHolder.java:533)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at com.nineoldandroids.animation.ObjectAnimator.initAnimation(ObjectAnimator.java:410)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at com.nineoldandroids.animation.ValueAnimator.setCurrentPlayTime(ValueAnimator.java:538)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at com.nineoldandroids.animation.ValueAnimator.start(ValueAnimator.java:928)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at com.nineoldandroids.animation.ValueAnimator.start(ValueAnimator.java:951)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at com.nineoldandroids.animation.ObjectAnimator.start(ObjectAnimator.java:385)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at com.nineoldandroids.animation.AnimatorSet.start(AnimatorSet.java:502)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at me.relex.circleindicator.CircleIndicator.createIndicators(CircleIndicator.java:146)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at me.relex.circleindicator.CircleIndicator.setViewPager(CircleIndicator.java:87)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at org.smssecure.smssecure.IntroScreenActivity.onCreate(IntroScreenActivity.java:97)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1072)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1836)
01-09 08:17:57.349 E/AndroidRuntime( 2896): 	... 11 more
01-09 08:17:57.369 W/ActivityManager( 1355):   Force finishing activity org.smssecure.smssecure/.IntroScreenActivity
```

--- a/src/org/smssecure/smssecure/IntroScreenActivity.java
+++ b/src/org/smssecure/smssecure/IntroScreenActivity.java
@@ -94,8 +94,17 @@ public class IntroScreenActivity extends BaseActionBarActivity {
 
     final int numberOfPages = introScreen.get().getPages().size();
     if (numberOfPages > 1) {
-      indicator.setViewPager(pager);
-      indicator.setOnPageChangeListener(new OnPageChangeListener(introScreen.get()));
+      try {
+        // For some reason this seems to throw an NPE on Android 2.3 - work around it for now
+        // See https://github.com/SMSSecure/SMSSecure/issues/311
+        indicator.setViewPager(pager);
+        indicator.setOnPageChangeListener(new OnPageChangeListener(introScreen.get()));
+      }
+      catch (NullPointerException e){
+        Log.i(TAG, "NPE when trying to setViewPager: " + e.toString());
+        onContinue();
+        return;
+      }
     } else {
       indicator.setVisibility(View.GONE);
     }

